### PR TITLE
[DESKTOP] Add mailserver and bootnodes settings

### LINKED ISSUE #7430: "Pick another mailserver" leads to white screen on desktop
### Description
_Type_: Bug

_Summary_: introduced in https://github.com/status-im/status-react/pull/7425. When you app can't connect to mailserver and "Error connecting to mailserver" - clicking on "Pick another" leads to white screen

#### Expected behavior
can pick another mailserver

#### Actual behavior
white screen

### Reproduction
[comment]: # (Describe how we can replicate the bug step by step.)

* Open Status
* Log in
* Try to get "Error connecting to mailserver" (constantly reproducible for me when I choose `mail03.ac-cn-hongkong-c.eth.beta`)
* Click on "Pick another"

### Additional Information

* Status version: any that contains https://github.com/status-im/status-react/pull/7425
* Video: https://take.ms/yugeK
* Operating System: Mac OS

#### Logs
[Status.log](https://github.com/status-im/status-react/files/2835951/Status.log)

### PR DESCRIPTION
**Pull request by vitvly**
Adds `Add mailserver` and `Bootnodes settings` buttons to Desktop's `Advanced settings` screen

Fixes #7430

### CODE REVIEW

File: src/status_im/ui/components/checkbox/view.cljs [hunk 1]
```diff
@@ -19,8 +19,10 @@
           (when on-value-change
             {:on-press #(on-value-change (not checked?))}))
    [react/view (styles/icon-check-container checked?)
-    (when checked?
-      [icons/icon :tiny-icons/tiny-check {:color colors/white}])]])
+    (if platform/desktop?
```
Comment (yenda) [inline src/status_im/ui/components/checkbox/view.cljs]: why is it different for desktop?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/components/checkbox/view.cljs (hunk 1 above)
Comment (vitvly) [reply src/status_im/ui/components/checkbox/view.cljs]: Desktop handles :color differently, without it icon is not visible.

**Comment by annadanchenko** [comment]
on Windows, build 4 buttons are narrow but they work
mail

**Review by yenda — APPROVED** [review]

**Comment by annadanchenko** [comment]
> on Windows, build 4 buttons are narrow but they work
> mail

I wouldn't fix it as mobile ui is coming

**Comment by churik** [comment]
Checked on Linux Ubuntu 18.04 and Mac Os:

- upgrade from current nightly
- adding custom bootnode, validation
- switching off \ on bootnodes
- switching to custom mailserver (working and non-working)
- that issue https://github.com/status-im/status-react/issues/7430 is fixed

**Pull request merged**